athena-synapse/athena-synapse.yaml

```diff
@@ -40,6 +40,14 @@ Parameters:
     Description: 'Lambda memory in MB (min 128 - 3008 max).'
     Default: 3008
     Type: Number
+  LambdaRole:
+    Description: "(Optional) A custom role to be used by the Connector lambda"
+    Default: ""
+    Type: String
+  PermissionBoundary:
```

Comment:
One last nit:
Is it possible to name this `PermissionsBoundary` to be consistent with how this is named normally?

If not (because the name conflicts), can you name it `PermissionsBoundaryARN`?

Similarly, if you end up naming it `PermissionsBoundaryARN` can you rename `LambdaRole` to `LambdaRoleARN` ?

Sorry for being nitpicky about this.

Reply:
Yes Henry, I will check on this.